cookbooks/aws-parallelcluster-config/files/default/cloudwatch_agent/write_cloudwatch_agent_json.py

```diff
@@ -11,6 +11,7 @@
 import os
```

Comment:
Can we add python files used for cloudwatch_agent to linters here: https://github.com/aws/aws-parallelcluster-cookbook/blob/develop/tox.ini#L17-L20?

Reply:
Yes, added and autoformat write_cloudwatch_agent_json.py